pkg/ccl/backupccl/backupbase/constants.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Licensed as a CockroachDB Enterprise file under the Cockroach Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     https://github.com/cockroachdb/cockroach/blob/master/licenses/CCL.txt
+
+package backupbase
+
+const (
+	// IncludeManifest is a named const that can be passed to FindPriorBackups.
+	IncludeManifest = true
+
+	// OmitManifest is a named const that can be passed to FindPriorBackups.
+	OmitManifest = false
```

Comment:
Since these two are only used as arguments to a function that lives in `backupdestination` perhaps we can move them there now.

Reply:
Currently IncludeManifest is used in `manifest_handling.go` which will move into `backupinfo`. OmitManifest is used `backupdest` as well as `cliccl`.

Now that I'm looking at the method `resolveBackupManifests` that uses `IncludeManifest` in `manifest_handling.go`, this will cause issues when moving `manifest_handling.go` into `backupinfo` since `backupinfo` should probably not depend on `backupdest`?

Reply:
Perhaps I misread, but the only uses of these are in calls to FindPriorBackups. Since FindPriorBackups is in `backupdest`, then anything that depends on these constants already depends on `backupdest`.  So, the only way to break the dependency is to move FindPriorBackups and then these constants can live in whatever package that function lives in.

All that said, you have this dependency graph more in mind than I do at the moment, and I think we can do this iteratively so don't treat this as blocking.

Reply:
Chatted offline, this is a dependency we are going to have to break at some point. Whether we do this through dependency injection/interfaces is still to be seen. I think we agree that we should draw the lines in the sand with these packages and then iterate on cleaning up the dependency graph. I might capture some of these improvements in a TODO list issue just so that we don't forget.